src/script/auth/component/AccountForm.tsx

```diff
@@ -239,17 +234,60 @@ const AccountForm = ({account, ...props}: Props & ConnectedProps & DispatchProps
         style={{justifyContent: 'center'}}
       >
         <CheckboxLabel>
-          <FormattedMessage
-            {...accountFormStrings.terms}
-            values={{
-              // eslint-disable-next-line react/display-name
-              a: (...chunks: any[]) => (
-                <a target="_blank" rel="noopener noreferrer" data-uie-name="go-terms" href={createURLForToU()}>
-                  {chunks}
-                </a>
-              ),
-            }}
-          />
+          {Config.getConfig().FEATURE.ENABLE_ACCOUNT_REGISTRATION_ACCEPT_TERMS_AND_PRIVACY_POLICY ? (
+            <FormattedMessage
+              {...accountFormStrings.termsAndPrivacyPolicy}
+              values={{
+                // eslint-disable-next-line react/display-name
+                privacypolicy: (...chunks: any[]) => (
```

Comment:
Are `chunks` React Elements or strings?

Reply:
Both ;P

Reply:
Great, please type them if you know what they are.